utils/commands.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from django.core.management import BaseCommand as DjangoBaseCommand
+
+
+class BaseCommand(DjangoBaseCommand):
+    logger = logging.getLogger(__name__)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="Dry run, don't delete any of the files or directories",
+        )
+
+    def setup_logging(self, verbosity):
```

Comment:
We do have an environment variable `DJANGO_LOG_LEVEL` which does control the log level, so this sort of overrides that setting. But then again controlling verbosity does sound normal like normal thing to be able to do on a management command. :shrug: :+1:

Reply:
Yeah, I think specifically for commands it's useful to be able to specify it.